src/Modules/CrestApps.OrchardCore.Subscriptions/Services/SubscriptionsAdminMenu.cs

```diff
@@ -50,9 +56,15 @@ protected override ValueTask BuildAsync(NavigationBuilder builder)
                 .AddClass("subscriptions")
                 .Id("subscriptions")
                 .Action(_subscriptionRouteValues)
-                .Permission(SubscriptionPermissions.ManageOwnSubscriptions)
+                .Permission(SubscriptionPermissions.ManageSubscriptions)
                 .LocalNav()
-            );
+            )
+            .Add(S["My Subscription"], S["My Subscription"].PrefixPosition(), subscriptions => subscriptions
```

Comment:
This will probably end up 

Subscriptions >> Manage 

And will be pointing to ManageSubscribtionsController which is for the admin not the subscribers.